.travis.yml

```diff
@@ -1,24 +1,18 @@
-sudo: required
-dist: trusty
-# Force travis to use its minimal image with default Python settings
+sudo: required 
+dist: trusty 
 language: generic
-compiler:
-  - gcc
-
+notifications:
```

Comment:
Well, if we remove this...I want it removed in all the other repositories, too
For the sake of consistency!